cvat/apps/engine/cache.py

```diff
@@ -195,21 +336,33 @@ def get_or_set_task_chunk(
         chunk_number: int,
         *,
         quality: FrameQuality,
-        set_callback: Callable[[], DataWithMime],
+        set_callback: Callable[..., DataWithMime],
+        set_callback_args: Union[list[Any], None] = None,
+        set_callback_kwargs: Union[dict[str, Any], None] = None,
```

Comment:
Consider adding a class for this triad:
```
@attrs.define
class Callback:
    callable: ...
    args: ...
    kwargs: ...
```

Then it can be just another type of the callable in the `Cache` methods. Inside you'll check the callback type. This should clean up the API of the class methods.